lib/services/servertech-obm-service.js

```diff
@@ -23,27 +23,27 @@ function serverTechObmServiceFactory(Promise, BaseObmService, _) {
     util.inherits(ServerTechObmService, BaseObmService);
 
     ServerTechObmService.prototype.reboot = function() {
-        return this._runInternal(this.mibPowerCommand + '.' + this.options.config.port, 'i', '3');
+        return this._runInternal([this.mibPowerCommand + '.' + this.options.config.port, 'i', '3']);
     };
 
     ServerTechObmService.prototype.powerOn = function() {
-        return this._runInternal(this.mibPowerCommand + '.' + this.options.config.port, 'i', '1');
+        return this._runInternal([this.mibPowerCommand + '.' + this.options.config.port, 'i', '1']);
     };
 
     ServerTechObmService.prototype.powerOff = function() {
-        return this._runInternal(this.mibPowerCommand + '.' + this.options.config.port, 'i', '2');
+        return this._runInternal([this.mibPowerCommand + '.' + this.options.config.port, 'i', '2']);
     };
 
     ServerTechObmService.prototype.powerStatus = function() {
         return this._runInternal(this.mibStatusCommand + '.' + this.options.config.port, 'snmpwalk')
         .then(function (result) {
-            if (_.contains(result.stdout.contains, 'INTEGER: on') ||
-                _.contains(result.stdout.contains, 'INTEGER: idleOn')) {
+            if (_.contains(result.stdout, 'INTEGER: on') ||
+                _.contains(result.stdout, 'INTEGER: idleOn')) {
```

Comment:
Are you sure result.stdout.contains is incorrect data path? If you are not sure about this, you should find the old devices and try on it. Otherwise, below lines are safer.

_.contains(result.stdout, 'INTEGER: on') ||
_.contains(result.stdout, 'INTEGER: idleOn' ||
_.contains(result.stdout.contains, 'INTEGER: on' ||
_.contains(result.stdout.contains, 'INTEGER: idleOn'

Reply:
ok，I will a old one to do some test